File: Mobile/SchedHoliday/app/src/main/java/be/helmo/schedholiday/repository/UserHoliday/DTOUserHoliday.kt
package be.helmo.schedholiday.repository.UserHoliday

import kotlinx.serialization.Serializable
import kotlinx.serialization.encodeToString
import kotlinx.serialization.json.Json

@Serializable
class DTOUserHoliday (
    val IdHoliday : List<String>,
    val IdUser : List<String>
){
    fun toJson() : String{
        return Json.encodeToString(this)
    }
}

File: API/SchedHoliday/Repo/UserHoliday/DTOUserHoliday.cs
using SchedHoliday.Repo.Holiday;
using SchedHoliday.Repo.User;

namespace SchedHoliday.Repo.UserHoliday
{
    public class DTOUserHoliday
    {
        public IEnumerable<string> IdHoliday {get; set;}
        public IEnumerable<string> IdUser { get; set; }
        public DTOHoliday DTOHoliday {get; set; }
        public DTOUser DTOUser { get; set; }

    }
}
